passphrase/passphrase.go

```diff
@@ -27,6 +27,9 @@ way to recover this key. You can find the key in your config directory.`
 )
 
 var (
+	// ErrEmpty is returned if the passphrase entered for a new key is empty, or white-space only
+	ErrEmpty = errors.New("Passphrase is empty")
```

Comment:
Wondering if these errors should be exported, or if it's ok to un-export them (as they're really specific to the default validator)

Reply:
This seems to be checked and returned before the validator is called, so it may make sense to export.